src/code/fv-components/breed-button.js

```diff
@@ -0,0 +1,31 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import t from '../utilities/translate';
+
+const BreedButtonView = React.createClass({
```

Comment:
Seems like this could have been implemented as a stateless functional component.